### DISCUSSION TITLE
when will the V3 come out?

### DISCUSSION THREAD

**Discussion by liuliang-wt** [elsa-workflows/elsa-core#4153] [Q&A]
Is there  a specific date？I'm hesitating whether to wait for V3 or stick with v2 .....

**Comment by sfmskywalker** [comment]
There's no exact date, just a guestimate. It largely depends on when the required features are done and issues have been resolved. We try to maintain an accurate overview [here](https://github.com/elsa-workflows/elsa-core/milestone/25).

Once RC2 ships, we'll give it another month or so to gather feedback. Barring any serious issues, that's when we'll release to NuGet.

Some users already use the preview packages from feedz.io, but it depends on how much risk you are comfortable taking, and what your own timeline is looking in terms of when you need a stable implementation.
